src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -376,14 +376,7 @@ internal BoundExpression BindToNaturalType(BoundExpression expression, BindingDi
                     break;
                 case BoundUnconvertedInterpolatedString unconvertedInterpolatedString:
                     {
-                        // We determine the best method of emitting as a string (either via Concat, Format, or the builder pattern)
-                        // during lowering, and it's not part of the publicly-visible API, unlike conversion to a builder type.
-                        result = new BoundInterpolatedString(
-                            unconvertedInterpolatedString.Syntax,
-                            unconvertedInterpolatedString.Parts,
-                            unconvertedInterpolatedString.ConstantValue,
-                            unconvertedInterpolatedString.Type,
-                            unconvertedInterpolatedString.HasErrors);
+                        result = BindUnconvertedInterpolatedStringToString(unconvertedInterpolatedString, diagnostics);
```

Comment:
nit: naming. We're not returning a string. Consider just `BindUnconvertedInterpolatedString`

Reply:
This naming is intentional. We're binding this interpolated string to `System.String`, not to some builder type. We'll eventually have a `BindUnconvertedInterpolatedStringToBuilderType` as well.

---
In reply to: [608901424](https://github.com/dotnet/roslyn/pull/52422#discussion_r608901424) [](ancestors = 608901424)